src/cmdlib.sh

```diff
@@ -65,10 +65,17 @@ else
     esac
 fi
 
+# cap qemu smp at some reasonable level to not exceed limitation of some platforms
+if [ "$(nproc)" -gt 16 ]; then
+    QEMU_PROCS=16
+else
+    QEMU_PROCS="$(nproc)"
+fi
 # only use 1 core on kubernetes since we can't determine how much we can actually use
-grep -q 'kubepods' /proc/1/cgroup && procs=1 || procs="$(nproc)"
-QEMU_KVM+=" -smp $procs"
+grep -q 'kubepods' /proc/1/cgroup && QEMU_PROCS=1
```

Comment:
See https://github.com/coreos/coreos-assembler/pull/700#discussion_r313606353, though in this case we *have* to switch it since the statement can fail now. Probably cleaner to just fold it into the if-else chain above?